cmd/itest/common_test.go

```diff
@@ -11,7 +11,9 @@ import (
 )
 
 func runSingle(t *testing.T, args ...string) error {
+	t.Log("running runSingle")
```

Comment:
Can we remove the log statements? I guess you put them in to verify that `t.Parallel()` was doing the right thing?